contrib/functions/go/delete-annotations/README.md

```diff
@@ -0,0 +1,34 @@
+# delete-annotations
+
+## Overview
+
+<!--mdtogo:Short-->
```

Comment:
Please ensure you are using these tags correctly. Please take a look at the template.

Reply:
Acked